### ISSUE TITLE
Use mariadb instead of mysql

### ISSUE THREAD

**Issue by arthurzenika** [CSSCorp/openstack-automation#14]
# featurerequest

(if we start using theses states in production I might contribute this if no one has since I posted this ticket)

**Comment by Akilesh1597** [comment]
I agree with this one. In fact we have to support all db that Openstack
itself supports. However this would mean a lot of changes.
1. Make the choice based on pillar data
2. Write new states for each of the db supported
3. Choose the state on the fly
I will start looking into this right away. It would be a good feature to
have.

**Comment by yashaga** [comment]
Have started working on this. We were studying this and think that we would be able to migrate MySQL to Mariadb.

What's the process of assigning this issue?
